tools/pattern-lab/config/config.yml

```diff
@@ -0,0 +1,79 @@
+v: 2.6.0
+overrideConfig: false
+ie:
+  - DS_Store
+  - less
+  - scss
```

Comment:
Trying to verify which of these keys (`ie` or `id`) prevents the copy of assets from source to public.

Reply:
This is updated to ignore our custom source folders.